PocketKit/Sources/PocketKit/Collections/CollectionStoryViewModel.swift

```diff
@@ -93,6 +95,12 @@ extension CollectionStoryViewModel: RecommendationCellViewModel {
     }
 
     var primaryAction: ItemAction? {
-        return nil
+        .recommendationPrimary { _ in
+            if !self.collectionStory.isSaved {
+                self.source?.save(collectionStory: self.collectionStory)
+            } else {
+            self.source?.archive(collectionStory: self.collectionStory)
```

Comment:
Ultra nit: Indentation
```suggestion
                self.source?.archive(collectionStory: self.collectionStory)
```

Reply:
Thanks for catching it!